### ISSUE TITLE
What are the available commands?

### ISSUE THREAD

**Issue by jondwillis** [umesan/homebridge-roomba#22]
I know "Hey Siri, start/stop Roomba" works, but what else? For instance, is there a command to dock the Roomba?

Can we add the available commands to the readme?

**Comment by takamasa-arch** [comment]
No, this version is only available to just on/off only. If you want to do more like dock the roomba, it's better to utilize ifttt/webhooks method. See my cases. (Use google translate)

https://simple-was-best.com/blog-entry-homebridge-ifttt-webhooks-roomba-siri.html
